enginetest/scriptgen/setup/scripts/integration_test

```diff
@@ -1131,3 +1131,1840 @@ update information_schema.statistics set cardinality = 36 where table_name = 'D3
 exec
 update information_schema.statistics set cardinality = 64 where table_name = 'ZTFET' and table_schema = 'mydb';
 ----
+
+exec
+CREATE TRIGGER SZW6V_on_insert BEFORE INSERT ON SZW6V
+FOR EACH ROW
+BEGIN
+  IF
+    NEW.T4IBQ IN (SELECT SVAZ4 FROM TPXHZ)
+    OR
+    NEW.V7UFH IN (SELECT SVAZ4 FROM TPXHZ)
+    OR
+    NEW.SYPKF IN (SELECT SVAZ4 FROM TPXHZ)
+    OR
+    NEW.H4DMT IN (SELECT SVAZ4 FROM TPXHZ)
+  THEN
+    -- SET @custom_error_message = (SELECT error_message FROM trigger_helper_error_message WHERE DZLIM = 'SVAZ4');
```

Comment:
Were these commented out in the original, or did you add that?

Reply:
comments are copied from the original source